eox_core/api/v1/views.py

```diff
@@ -137,6 +137,21 @@ def get(self, request, *args, **kwargs):
 
         return Response(response_data)
 
+    def patch(self, request, *args, **kwargs):
+        """
```

Comment:
could you add an example with the url type of request and data which are required

Reply:
Yes, I'll do it.